metaflow/plugins/argo/argo_workflows.py

```diff
@@ -954,6 +934,92 @@ def _container_templates(self):
                 )
             )
 
+            if node.name == "end":
+                if self._ignore_events:
+                    return
+                elif are_events_configured():
+                    yield self._make_lifecycle_hook_container_template(
+                        self._make_env(node), "succeeded"
+                    )
+                    yield self._make_lifecycle_hook_container_template(
+                        self._make_env(node), "failed"
+                    )
+
+    def _make_lifecycle_hook_container_template(self, env, status):
+        t = LifecycleHookContainerTemplate(current_flow_name(), status)
+        t.env(env)
+        return t
+
+    def _make_env(self, node, retry_count=None):
```

Comment:
It doesn't seem like any of these env variables are needed by the docker container responsible for emitting the event (besides flow name and run id). 

Currently, that docker container is responsible for 
1. emitting the metadata (which we can move to the triggered workflow as part of a decorator lifecycle - take a look at argo_workflows_decorator)
2. emitting the actual event (we should explore embedding the logic within the entry point in itself to save an extra image's hurdle)